From 53a24086a397b111f04b10bbc3056a95a2c041a6 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Fri, 23 Nov 2012 13:37:45 +0800
Subject: [PATCH 368/505] ENGR00234665 Enable pxp library test program
 building

Build the pxp library test program on mx6sl

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 test/pxp_lib_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/pxp_lib_test/Makefile b/test/pxp_lib_test/Makefile
index 20a9e41..c6895a4 100644
--- a/test/pxp_lib_test/Makefile
+++ b/test/pxp_lib_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX50 IMX51 IMX5 IMX6Q
+INCLUDE_LIST:=IMX50 IMX51 IMX5 IMX6Q IMX6S
 
 SRCS = pxp_test.c utils.c
 CFLAGS += -lpxp -lpthread
-- 
1.8.0

